src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -195,4 +276,79 @@ private String getAddress(Principal principal) {
     public Multi<Facet> getBundleFacets(@Context SecurityContext sec) {
         return bundleResources.getBundles().onItem().transform(b -> new Facet(b.getId().toString(), b.getName(), b.getDisplayName()));
     }
+
+    @DELETE
+    @Path("/eventTypes/{eventTypeId}/mute")
+    @Operation(summary = "Mute an event type, removing all its link with behavior groups.")
+    @RolesAllowed(RbacIdentityProvider.RBAC_WRITE_NOTIFICATIONS)
+    public Uni<Boolean> muteEventType(@Context SecurityContext sec, @PathParam("eventTypeId") UUID eventTypeId) {
+        return getAccountId(sec)
+                .onItem().transformToUni(accountId -> behaviorGroupResources.muteEventType(accountId, eventTypeId));
+    }
+
+    @POST
+    @Path("/behaviorGroups")
+    @Operation(summary = "Create a behavior group.")
+    @RolesAllowed(RbacIdentityProvider.RBAC_WRITE_NOTIFICATIONS)
+    public Uni<BehaviorGroup> createBehaviorGroup(@Context SecurityContext sec, @NotNull @Valid BehaviorGroup behaviorGroup) {
+        return getAccountId(sec)
+                .onItem().transformToUni(accountId -> behaviorGroupResources.create(accountId, behaviorGroup));
+    }
+
+    @PUT
+    @Path("/behaviorGroups/{id}")
+    @Operation(summary = "Update a behavior group.")
+    @RolesAllowed(RbacIdentityProvider.RBAC_WRITE_NOTIFICATIONS)
+    public Uni<Boolean> updateBehaviorGroup(@Context SecurityContext sec, @PathParam("id") UUID id, @NotNull @Valid BehaviorGroup behaviorGroup) {
+        return getAccountId(sec)
+                .onItem().transformToUni(accountId -> {
+                    behaviorGroup.setBundleId(id);
+                    return behaviorGroupResources.update(accountId, behaviorGroup);
```

Comment:
As is, would this allow anyone to take over other's `BehaviorGroup` ?

I think we should verify if `BehaviorGroup.accountId == accountId`.